src/main/java/com/scalefocus/blog_api/controller/BlogController.java

```diff
@@ -49,6 +49,12 @@ public ResponseEntity<BlogDto> addTagToBlog(@PathVariable Long blogId, @RequestB
     public ResponseEntity<BlogDto> deleteTagFromBlog(@PathVariable Long blogId, @PathVariable Long tagId){
         return new ResponseEntity<>(blogService.removeTag(blogId, tagId), HttpStatus.OK);
     }
+
+    //users can get all blogs by specific tagName
+    @GetMapping("/tagName")
+    public ResponseEntity<List<BlogDto>> getAllBlogsByTagName(@RequestParam String tagName){
```

Comment:
Consider to use @PathVariable instead. URLs with query parameters may be less expressive and slightly harder to read than clean path-based URLs
Also in our case we are using only one param, where the power of RequestParm is when used for multiple filter example: ?tagName=tech&sortBy=date

Reply:
Done